### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by tifa365** [StatePol/Database#1]
First of all, big congratulations for completing this project! I think you filled a huge research gap here and the dataset will be quite valuable in the future. Next to the field of political science I can also imagine data journalists from regional newspaper wil find this  a helpful repository for the future.

I have some comments regarding the webpage, not really bugs, but really just some statement what was my experience using it. 

User Interface:

- "Application" in the menu to me sounded more like different ways to use the data for other projects, rather than a shiny app. Maybe a more fitting title would be just "App" or, more old-fashioned, "Interactive App"? I think for most people this would be easier to understand at first glance.
- I'm a fan of dark mode but after activating, I could only experience it in the "Team" section - nowhere else. Perhaps something in the Quarto settings?

Shiny App:

- Are you planning on releasing the code? I was curious and wanted to have a look, but I could not find it in the repo. Or did I simply miss it?

Programming Language:

- Quite clear you concentrated on R rather than Python, but I'd love to see some basic starter code for Python as well. It's not pretty but it does the job https://github.com/tifa365/statepol_python/blob/main/download_instructions_python.ipynb

**Comment by danielkuhlen** [comment]
Thank you very much for all the comments and suggestions; they were really helpful!

**User Interface**
We changed the name to Interactive App. The dark mode option was due to a bug in the code we had not yet corrected; it didn't really work for the website, so we decided to remove it.

**Shiny App**
Yes, we will release the code within the next few weeks. As this is an ongoing project, the Shiny app is not yet finalized. As soon as it is, the respective repository will be made public.

**Programming Language**
Thank you very much for providing this code! We adapted it slightly and added it to the download section.
 
Again, thanks for these suggestions - they definitely improved the website. We hope you enjoy using the data!
